client/package.json

```diff
@@ -1,17 +1,56 @@
 {
-  "name": "trypurescript-client",
-  "private": true,
+  "name": "tps",
+  "private": "true",
   "config": {
     "configpath": "config/dev/*.purs"
   },
   "scripts": {
-    "clean": "rimraf output",
-    "build": "spago bundle-app --path $npm_package_config_configpath --purs-args '--censor-lib --strict' --to public/js/index.js"
+    "c-user-facing": "# --------- user facing scripts -------",
+
+    "c-gen-css": "# -- Generate tailwind css AND purs css wrapper",
+    "gen-css": "npm run build-css-only && npm run css2purs",
+
+    "c1-lock-css": "# -- Strip away unused css from autogenerated purs files.",
+    "c2-lock-css": "# -- This improves rebuild times, but won't allow adding new css.",
+    "c3-lock-css": "# -- So re-run gen-css before making additional css changes.",
+    "lock-css": "npm run bundle && npm run css-purge && npm run css2purs",
+
```

Comment:
It turns out that `npm i` will reformat this file and remove the unnecessary newlines I added for readability. Really terrible that you can't add comments to json files without workaround like this. Would like to migrate to something better if it exists.